examples/.gitignore

```
@@ -2,3 +2,4 @@
 /viame-example/**
 **/imagenet_classes.txt*
 /*.jpg
+/scikit-learn-exampel/**
```

`example` is mispelled, and can you move this up to where the other example directories are defined?

```suggestion
/scikit-learn-example/**
```